### ISSUE TITLE
[BUG] decodeEncodeEntities does not work

### ISSUE THREAD

**Issue by JokerQyou** [lublak/n8n-nodes-text-manipulation#6]
### Describe the bug

When using decode/encode entities feature, the node always errors with "ERROR: url, xml, html or nothing are valid options".

### Describe the expected behavior

decode and encode should work without any issue.

### What is your Node.js version?

unknown

### What is your n8n version?

0.216.0

### What is your n8n-nodes-text-manipulation version?

v1.0.5

### What operating system are you seeing the problem on?

Linux

### Operating system version (or if other, then please fill in complete name and version)

irrelevant

### Relevant log output

```shell
I've got no logs but these lines:


Stack:

NodeOperationError: url, xml, html or nothing are valid options
    at Object.execute (/root/.n8n/nodes/node_modules/n8n-nodes-text-manipulation/nodes/TextManipulation/TextManipulation.node.ts:1266:29)
    at Workflow.runNode (/usr/local/lib/node_modules/n8n/node_modules/n8n-workflow/dist/Workflow.js:658:51)
    at /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/WorkflowExecute.js:585:68
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

This issue is caused by a typo [here](https://github.com/lublak/n8n-nodes-text-manipulation/blob/2ca0f31ec01a9e61ad34b1739133a9a02395f8b0/nodes/TextManipulation/TextManipulation.node.ts#L1227).

If you could just replace the word `manipulation.decodeWithEntitie` with `manipulation.decodeWithEntities` it would probably work. Because `manipulation.decodeWithEntitie` is always `undefined` so the switch statements jumps to `default`, thus the above error stack.

**Comment by lublak** [comment]
@JokerQyou thanks for the report :) i will fix this!
I'll get to it tomorrow or tonight.

**Comment by lublak** [comment]
released in: 1.2.0

**Comment by lublak** [comment]
@JokerQyou released in: 1.2.0
